src/main/kotlin/com/hibob/ex2-loops.kt

```diff
@@ -0,0 +1,37 @@
+package com.hibob
+
+fun isContainDolarSymbol(s: String): Boolean = "$" !in s
```

Comment:
The meaning here is actually "dollar sign not contained in string" 😄

Reply:
updated